src/test/java/com/epam/definitions/ReflectionDefinitions.java

```diff
@@ -22,90 +17,49 @@
     private Object object;
 
     @Given("Class name - {string}")
-    public void intClass(String className) {
-
-        try {
-            clazz = Class.forName(className);
-            object = clazz.getConstructor().newInstance();
-        } catch (ClassNotFoundException | NoSuchMethodException | IllegalAccessException | InstantiationException | InvocationTargetException e) {
-            e.printStackTrace();
-        }
+    public void intClass(final String className) {
 
+        clazz = getClazz(className);
+        object = getNewInstance(clazz);
     }
 
     @Then("Verify if current Object.getName equals {string}")
```

Comment:
1. You message says "Object.getName", but actually you call Class.getName.
However I would recommend you to rename it like

"Verify initialized object has expected {string} class name"